terraform/aws/analytical-platform-data-engineering-sandbox-a/github-actions-roles/locals.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  eks_oidc_id  = "801920EDEF91E3CAB03E04C03A2DE2BB" # AP Compute Production
+  eks_oidc_url = "https://oidc.eks.eu-west-2.amazonaws.com/id/${local.eks_oidc_id}"
+  account_ids = {
```

Comment:
Can I ask you to move account IDs and Tags into a `variables.tf` + `terraform.tfvars` setup so that we can easily vary these if we have a number of different users on this in future - This will help standardise this with the rest of our repo